lambda/service/store_dynamodb/store_dynamodb.go

```diff
@@ -26,15 +27,18 @@ func NewAccountDatabaseStore(db *dynamodb.Client, tableName string) DynamoDBStor
 func (r *AccountDatabaseStore) Insert(ctx context.Context, account Account) error {
 	item, err := attributevalue.MarshalMap(account)
 	if err != nil {
+		log.Printf("couldn't perform insert. Here's why: %v\n", err)
 		return err
 	}
 	_, err = r.DB.PutItem(context.TODO(), &dynamodb.PutItemInput{
```

Comment:
This should be the `ctx` that was passed into `Insert` not a new context.

Reply:
Updated, thanks